cfgov/unprocessed/apps/paying-for-college/js/college-costs/views/school-view.js

```diff
@@ -131,9 +130,9 @@ const schoolView = {
         'INPUT[name="' + name + '"][value="' + value + '"]'
       );
       if (input !== null) {
-        const label = closest(input, '.m-form-field__radio').querySelector(
-          'LABEL'
-        );
+        const label = input
+          .closest('.m-form-field__radio')
+          .querySelector('LABEL');
```

Comment:
I'd leave it for now to keep this PR in scope, but I wonder if this could be compacted to `.closest('.m-form-field__radio LABEL')`.